chapters/intro/intro.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+project: 000
+title: Introduction
+chapter: true
+---
+
+# Introduction
+
+**[Arie van Deursen] and [Rogier Slag].**<br/>
+*Delft University of Technology, The Netherlands*
+
+[arie van deursen]: http://avandeursen.com
+[rogier slag]: https://github.com/rogierslag
+
+
+
+_Delft Students on Software Architecture_ is a collection of architectural descriptions of open source software systems written by students from Delft University of Technology during a master-level course taking place in the spring of 2015.
+
+At the start of the course, teams of 3-4 students could adopt a project of choice on GitHub. The projects selected had to be sufficiently complex and actively maintained (one or more pull requests merged per day).
+
+During a 10 week period, in which the students spent one third of their time on this course, the students engaged with these systems, in order to understand and describe their software architecture.
```

Comment:
I'm a big fan of the [Oxford Comma](http://en.wikipedia.org/wiki/Serial_comma) for this kind of sentences. But it's up to you whether to use this or not (as long as it's consistent I'm okay with either).